src/main/java/seedu/address/logic/parser/SortCommandParser.java

```diff
@@ -0,0 +1,48 @@
+package seedu.address.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.Comparator;
+import java.util.Optional;
+
+import seedu.address.commons.core.item.Item;
+import seedu.address.logic.commands.SortCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parser to create a sort command with the right arguments
+ */
+public class SortCommandParser {
+    public static final String DESCRIPTION = "desc";
+    public static final String PRIORITY = "priority";
```

Comment:
do you want to consider shortening this? i think `sort priority` is a little long, even though it is clear. maybe offer option for a short form, like `pri`? i believe we don't use any similar words in the app, so there shouldn't be ambiguity

Reply:
Hmm, you are right! I think I shall keep the long form but at the same time I will add in a short form variant!